i18n/locales/th-TH/hourofcode/th.yml

```diff
@@ -241,20 +243,21 @@ th:
   teacher_forum_desc: เยี่ยมชมฟอรัมผู้สอนชั่วโมงแห่งรหัสเพื่อรับคำแนะนำ ข้อมูลเชิงลึกและการสนับสนุนจากนักการศึกษาคนอื่น
   required_materials: 'อุปกรณ์ที่จำเป็น:'
   optional_materials: 'อุปกรณ์เสริม:'
-  for_students: สำหรับนักเรียน
+  for_students: For students
   for_educators: สำหรับครู
-  call_to_action_host_hoc: จัดชั่วโมงแห่งรหัส
+  call_to_action_host_hoc: Host an Hour of Code
   call_to_action_hold_an_event: จัดกิจกรรม
   call_to_action_register_event: ลงทะเบียนกิจกรรมของคุณ
   call_to_action_register_your_hoc: ลงทะเบียน Hour of Code ของคุณ
   call_to_action_select_activity: เลือกกิจกรรม
   call_to_action_start_activity: เริ่มกิจกรรม
   call_to_action_get_certificates: รับใบรับรอง
   call_to_action_go_beyond_hoc: ไปให้ไกลกว่า Hour of Code
-  call_to_action_explore_all_hoc_activities: สำรวจกิจกรรมHour of Code ทั้งหมดจาก Code.org
-  call_to_action_start_dance_party: เริ่มปาร์ตี้เต้นรำ
-  call_to_action_start_part_02: เริ่มตอนที่สอง
-  call_to_action_start_unplugged_activity: เริ่มกิจกรรมที่ถอดปลั๊ก
+  call_to_action_explore_all_hoc_activities: Explore all Hour of Code activities from
```

Comment:
This strings are regressing to English :(

Reply:
Never mind, I saw the translations were actually canceled in the Crowdin project. There are already new translations that will be picked up in the next sync.